test/coverage.js

```diff
@@ -120,8 +120,7 @@ describe('Coverage', () => {
         });
 
         expect(missedLines).to.include([
-            { filename: './while.js', lineNumber: '5', originalLineNumber: 11 },
-            { filename: './while.js', lineNumber: '6', originalLineNumber: 12 }
+            { filename: 'while.js', lineNumber: '3', originalLineNumber: 8 }
```

Comment:
The fact that only one missing line is detected is a possible bug.

I've replaced the `sourcemaps-inline.js` with a newer version processed by babel, and compacted into a single line:
```javascript
'use strict';// Load modules
// Declare internals
const internals={};exports.method=function(value){while(value){value=false;}return value;};
//# sourceMappingURL=data:application/json;base64,...
```
Coverage detects that `while(value)` is always false, but doesn't detect that `value=false` is never reached.

This only happens when generated file is compacted, I'll dig that a little more.